OMEdit/OMEditLIB/Simulation/SimulationDialog.cpp

```diff
@@ -806,8 +806,10 @@ void SimulationDialog::initializeFields(bool isReSimulate, SimulationOptions sim
               }
               i++;
             }
-          } else if ((simulationFlag.compare("reconcile") == 0) || (simulationFlag.compare("reconcileBoundaryConditions") == 0)) {
-            mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm(simulationFlag);
+          } else if (simulationFlag.compare("reconcile") == 0)  {
+            mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm("dataReconciliation");
+          } else if (simulationFlag.compare("reconcileBoundaryConditions") == 0) {
```

Comment:
Use `QStringLiteral("reconcile")` and `QStringLiteral("reconcileBoundaryConditions")`.